### ISSUE TITLE
[Rule Suggestion] Rewriting of rule 2.1

### ISSUE THREAD

**Issue by Highlander214** [EarthMC/Issue-Tracker#983]
As it stands rule 2.1 is confusing for both players and staff. The amount of times i can count having to tell players that trapping is allowed under certain conditions is just insane. I have taken the liberty of rewriting 2.1 to better fit the current meta of EMC.

"2.1 Trapping
The act of N/T Spawn trapping is forbidden as it ruins gameplay for many players. Traps that do not allow players to TP out of without death or intervention by a staff member are also forbidden. Traps that kill players outside of N/T spawns are allowed. If your are found to be using an illegal trap staff may issue warnings or even temp bans."

Feedback on this would be greatly appreciated

**Comment by AmphireYT** [comment]
I made an issue on changing this rule but Really nothing happened... will this rule will ever change and be illegal to trap kill?

**Comment by ghost** [comment]
i wish for that to be the case, i think N/T spawn trapping is just a scummy thing to do and ruins the game for those involved, im not saying outlaw scamming as a whole.

**Comment by Scorpionzzx** [comment]
Yea scamming is 1 thing as it's your choice to do a sketchy trade deal and your choice alone. But spawn traps can misleading ex: "Selling items at our shop in /n spawn sui" then a player goes there not knowing its a spawn trap and losing his items or worse if its a noob they lose gold because we all know they be dum and carry like 60g on them cuz they never heard of an ender chest.

**Comment by gabeeebaby** [comment]
Instead of banning trapping, sending multiple msgs advertising a trap should be prohibited. PVP may be turned on in plots for other reasons, such as killing someone who won't leave or fighting an enemy. While it is toxic to spawn trap, it would be hard to enforce and there would be backlash from many players.

**Comment by AmphireYT** [comment]
lets do a vote I think that a pretty good option considering many people have different views of trap killing. I really do like Scorpionzzx view about how his could be misleading. Also, it would be pretty nice to add lets say a warning to warn a player that this is a spawn trap (or known to be) and could lose everything. Im not saying put this idea in, (eartmc scams is pretty useful discord really like it) Im just saying that like if the mods/admins/fix doesnt make spawn trap somehow bannable

**Comment by ghost** [comment]
i agree with Scorpionzzx, there is a big difference between scamming and tp trapping,
